tests/Maker/MakeCommandTest.php

```diff
@@ -26,8 +26,12 @@ protected function getMakerClass(): string
     public function getTestDetails(): \Generator
     {
         yield 'it_makes_a_command_no_attributes' => [$this->createMakerTest()
-            ->addRequiredPackageVersion('symfony/console', '<5.3')
             ->run(function (MakerTestRunner $runner) {
+                if (\PHP_MAJOR_VERSION < 8) {
+                    // @legacy Remove when PHP 7 is now longer supported.
+                    $this->markTestSkipped('Test only runs on PHP < 8');
+                }
```

Comment:
There is a `setRequiredPhpVersion()` - would it work here?

Reply:
good catch again